public/locales/en/index.ts

```diff
@@ -0,0 +1,9 @@
+import address from './address.json'
```

Comment:
don't think it really makes sense to have a typescript file in the public directory